content/ecosystem/sdks/sdks.toml

```
@@ -569,3 +569,16 @@ featured_in = []
 description = """
 Matrix client library for MicroPython.
 """
+
+[[sdks]]
+name = "Matrix.swift"
+maintainer = "cvwright"
+maturity = "Alpha"
+language = "Swift"
+license = "Apache-2.0"
```

```suggestion
licence = "Apache-2.0"
```

Those pesky Brits got you with en_GB 😄

---

@cvwright do you want me to commit this change and merge your PR, or would you rather do it yourself?